.github/workflows/ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+  ci:
+    name: >-
+      ${{ matrix.os }} ${{ matrix.ruby }}
+
+    runs-on: ${{ matrix.os }}
+    if: |
+      !(   contains(github.event.pull_request.title,  '[ci skip]')
```

Comment:
So a PR with `"[ci skip]"` or `"[skip ci]"` will not trigger a build?

Reply:
Yes.  Pretty common, I think Appveyor and/or Travis may have had the behavior by default.

I can remove?